README.md

```diff
@@ -2,27 +2,30 @@
 
 > The `create-react-app` version of building docusaurus websites
 
-This is the official Docusaurus theme for balena docs. [Docusaurus 2](https://docusaurus.io/)is a modern static website generator that we used to generate docs. This repository hosts a customized docusarus theme and a docusaurus builder GitHub action. 
+This is the official Docusaurus theme for balena docs.
```

Comment:
Are you running an auto linter on markdown? It feels like it's adding a lot of changes that aren't relevant to the PR. Do reset them and add only the local preview section

Reply:
I like my auto linting :P

Reply:
But, the git history won't like the noise when that shows up on blame. To have a clean git history I recommend, linting these preferably in a different PR